src/cms-content/articles/index.ts

```diff
@@ -16,20 +20,28 @@ interface ArticleJSON extends ArticleMain {
 }
 
 export interface Article extends ArticleMain {
-  date: Date;
+  date: string;
 }
 
 function sanitizeArticle(article: ArticleJSON): Article {
   return {
     ...article,
     articleID: sanitizeID(article.articleID),
-    date: moment(article.date).toDate(),
+    date: moment(article.date).format('MMM Do, YYYY'),
```

Comment:
Consider setting `null` for optional values here so it's easier to work with them in their corresponding pages

Reply:
echoing comment above- might remove those fields altogether. tbd

Reply:
update: keeping for the time being